From 29f8fcfe816b83b3d5b1faca7b665b9b75e29497 Mon Sep 17 00:00:00 2001
From: D <dg@dg.com>
Date: Fri, 2 Feb 2018 06:41:17 -0800
Subject: [PATCH] PLease include a useful commit message!

---
 Jenkinsfile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Jenkinsfile b/Jenkinsfile
index 7b2b05d..ea0a3c3 100644
--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -1,4 +1,4 @@
 #!groovy
 node {
-echo 'hello from master Pipeline'
+echo 'Hello from master Pipeline'
 }
-- 
2.10.1 (Apple Git-78)

